### ISSUE TITLE
Server settings do not resolve Windows' hosts file

### ISSUE THREAD

**Issue by koma-cute** [Difegue/Tsukihi#6]
Setting the server settings to a domain defined only in the Windows hosts file results in a "TypeError: NetworkError when attempting to fetch resource". Visiting the domain in browser works fine, as does using the IP in Tsukihi.

Example hosts:
```
10.1.1.1 lrr.lan
```

**Comment by Difegue** [comment]
I think this is sadly more of a browser problem, the extension itself only uses very standard JS to negotiate connection to the server; I'm not sure why connections emitted by an extension would bypass the hosts file somehow.  

Does that happen on every browser? Haven't tried it myself.

**Comment by koma-cute** [comment]
Not really a huge deal I guess, just the behavior was a bit weird. It doesn't seem to happen on Chrome - I was using Firefox.

**Comment by Difegue** [comment]
Thinking back on it, you might have DoH enabled on FF, which does not go through the hosts file iirc: https://stackoverflow.com/questions/37452361/why-is-my-hosts-file-entry-being-ignored-by-the-browser

**Comment by koma-cute** [comment]
I do, but disabling it doesn't fix the issue.

**Comment by LuanaNight** [comment]
Any update on this, would be cool if this extension could be used on firefox

**Comment by LuanaNight** [comment]
Got it to work by downgrading to 1.2, 2.0 seems to break firefox

**Comment by Difegue** [comment]
Yeah I've had multiple reports over the years of the Manifest V3 version not working that well on Firefox; I personally still run 1.2 myself so 2.0 is really mostly for Chromiums.  

I really don't want to maintain both MV2 and MV3 versions, although the extension changes so rarely it might not really be that much of an issue.

**Comment by LuanaNight** [comment]
Yeah, v3 sucks, if 1.2 doesn't break then it's fine
